packages/utilities/psammead-locales/moment/ar.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable func-names */
+const moment = require('moment');
+require('moment/locale/ar');
+
+// these keys have been overridden to maintain western numerals
+// some other keys have been left as set upstream
+// see https://github.com/moment/moment/blob/develop/src/locale/ar.js
+
+const months = [
+  'يناير/ كانون الثاني',
+  'فبراير/ شباط',
+  'مارس/ آذار',
+  'أبريل/ نيسان',
+  'مايو/ أيار',
+  'يونيو/ حزيران',
+  'يوليو/ تموز',
+  'أغسطس/ آب',
+  'سبتمبر/ أيلول',
+  'أكتوبر/ تشرين الأول',
+  'نوفمبر/ تشرين الثاني',
+  'ديسمبر/ كانون الأول',
+];
+const symbolMap = {
+  '1': '1',
+  '2': '2',
+  '3': '3',
+  '4': '4',
+  '5': '5',
+  '6': '6',
+  '7': '7',
+  '8': '8',
+  '9': '9',
+  '0': '0',
+};
+
+moment.updateLocale('ar', {
```

Comment:
I'm not clear on what `postformat` does but this implementation produces the desired output without symbolMap

```
 /* eslint-disable func-names */
const moment = require('moment');
require('moment/locale/ar');

// these keys have been overridden to maintain western numerals
// some other keys have been left as set upstream
// see https://github.com/moment/moment/blob/develop/src/locale/ar.js

const months = [
  'يناير/ كانون الثاني',
  'فبراير/ شباط',
  'مارس/ آذار',
  'أبريل/ نيسان',
  'مايو/ أيار',
  'يونيو/ حزيران',
  'يوليو/ تموز',
  'أغسطس/ آب',
  'سبتمبر/ أيلول',
  'أكتوبر/ تشرين الأول',
  'نوفمبر/ تشرين الثاني',
  'ديسمبر/ كانون الأول',
];

moment.updateLocale('ar', {
  months,
  monthsShort: months,
  postformat(string) {
    return string.replace(/,/g, '،');
  },
});

```

Reply:
Had  to use symbolMap to replace the eastern arabic numerals with western numerals but tried this implementation seems to produce the desired effect too

Reply:
I'm uneasy with this solution for numerals as a) it's cryptic and b) could have unintended effects since `postformat is _called after formatting on every string_ (https://momentjs.com/docs/).

Reply:
for now the only effect `postformat` has is to replace the coma with the arabic coma,
I have counter checked with `ur` locale which has almost our intended output as for `ar` and seems i has no other effect https://github.com/moment/moment/blob/13a61b285c095bda7ea8e33156090ea5ccfeaef1/locale/ur.js#L83

Reply:
`postformat` function returns the western numerals unless overridden as was the case initially in the `ar` locale
```
postformat: function (string) {
        return string.replace(/\d/g, function (match) {
            return symbolMap[match];
        }).replace(/,/g, '،');
    },
```
which in this case `symbol map` mapped the western numerals to arabic numerals.
So basically we are overriding  the postformat from the base locale to use the western numerals